components/ecu/ecu_firmware/Core/Src/can.c

```diff
@@ -4,21 +4,38 @@
  * 
  * @date 2021/01/30
  * @author Blake Shular (blake-shular)
- */
+
+*/
 
 #include "can.h"
+#include "main.h"
+
+#define CAN_ID 0x450
+#define MESSAGE_LENGTH 3
+
 
 CAN_TxHeaderTypeDef current_message_header = {
-    0x624, 0, CAN_ID_STD, CAN_RTR_DATA, 6, DISABLE
+    CAN_ID, 0, CAN_ID_STD, CAN_RTR_DATA, MESSAGE_LENGTH, DISABLE
 }; 
 
-void CAN_send_current(int currentVal) {
+/*
+ * @param currentVal    The current flowing through the ECU, in Amps.
+                        The range is ±75A. 
+                        Scaling factor for this range: (1A/100)*(127/75A) = 127/7500
```

Comment:
Couple things:
- It's not current flowing through the ECU, it's the net current charging or discharging the battery pack on the high voltage bus
- The input to the function is not in amps, it's in 100ths of an amp, hence the scale factor. You're supposed to describe the function input here, not the output.
- I'd suggest putting the line about the scaling factor in a regular comment within the function